src/posts/2020-06-08-going-remote.md

```diff
@@ -0,0 +1,110 @@
+
+On the other hand, **when you are remote, you don't have anyone to talk with**. No water cooler. No coworker casually drinking from the beer fountain your eccentric startup CEO just acquired - *"so what if it's 11 AM on a Wednesday, it's 5'o clock somewhere!"* - When you're remote, you're on your own.
+
+.
+
+.
+
+.
+
+.
+
+.
+
+*"That sucks"* you conclude. *"This sucks"* you insist. *"Let's do dailies!"*
+
+**Daily meetings are the most common solution to combat isolation**. Schedule a time, get everyone online, have a chat with your team. If they are company-wide dailies (very common in startups) this might not apply to you. If, however, you are part of a small team within a larger team, you might end up in this scenario.
+
+The following is an excerpt of an internal Basecamp post I wrote last year, discussing being remote and how we can improve it:
+
+*I only talk with people once a day, during the daily. Literally just with Pedro. Only person from Subvisual. Other than him, the other participants not even part of the company. We mostly complain about stuff, Pedro's house not being finished yet and we casually mention what's on our plates for the day. That's it. A 15 min affair of communicating with human beings. There's nothing on Slack and, as far as I can tell, even the company seems closed. Fridays are very much office-centric (again, not that this is negative, at all). The best feedback you get is an occasional tweet with people jollying around the office. From my personal stand-point, I don't talk with anyone at all and my days are mostly spent reading or making things up to entertain my mind.*
+
+There's nothing wrong with dailies per se, but you should be aware - **they are a tool for project productivity, not for combating isolation**. This happened during a time when most of the team was in-office and all the goofing and joking around happened within it. Jokes wouldn't be on Slack. Nobody even would say anything on #random and if I took the initiative, I would barely get someone replying and would feel ignored and isolated. And afterwards I'd see everyone happy on Twitter. This is daunting. This is scary. And it leads to a negative feedback loop because after you neglect them, they will feel isolated and reduce their interactions, leading them to be even more neglected by other coworkers and increasing their isolation. **Applications like Slack, Basecamp or whatever, are the main method your coworkers have to communicate with you. Don't leave them hanging**. That's your tool to combat isolation, not dailies. Dailies are work, not watercooler, beer sipping, chess bashing fun times.
```

Comment:
_combating isolation_ sounds off, I think it should be _fighting isolation_.

> And it leads to a negative feedback loop because after you neglect them
Who's _them_? It's not clear from the context, hard to deduce.

```suggestion
There's nothing wrong with dailies per se, but you should be aware - **they are a tool for project productivity, not for fighting isolation**. This happened during a time when most of the team was in-office and all the goofing and joking around happened within it. Jokes wouldn't be on Slack. Nobody would even say anything on #random and if I took the initiative, I would barely get someone replying and would feel ignored and isolated. And afterwards I'd see everyone happy on Twitter. This is daunting. This is scary. And it leads to a negative feedback loop because after you neglect them, they will feel isolated and reduce their interactions, leading them to be even more neglected by other coworkers and increasing their isolation. **Applications like Slack, Basecamp or whatever, are the main method your coworkers have to communicate with you. Don't leave them hanging**. That's your tool to combat isolation, not dailies. Dailies are work, not watercooler, beer sipping, chess bashing fun times.
```